monkey/monkey_island/cc/resources/auth/register.py

```diff
@@ -23,20 +24,24 @@ def __init__(self, authentication_service: AuthenticationService):
 
     def post(self):
         """
-        Registers a new user
+        Registers a new user using flask security register
 
-        Gets a username and password from the request sent from the client,
-        and registers a new user
-
-        :raises InvalidRegistrationCredentialsError: If username or password is empty
-        :raises AlreadyRegisteredError: If a user has already been registered
         """
-
-        username, password = get_username_password_from_request(request)
-
         try:
-            self._authentication_service.register_new_user(username, password)
-            return make_response({"error": ""}, HTTPStatus.OK)
-        # API Spec: HTTP status code for AlreadyRegisteredError should be 409 (CONFLICT)
-        except (InvalidRegistrationCredentialsError, AlreadyRegisteredError) as e:
-            return make_response({"error": str(e)}, HTTPStatus.BAD_REQUEST)
+            username, password = get_username_password_from_request(request)
```

Comment:
Missing a check whether a user is already registered

Reply:
Nope. It is missing user limitation. If i try to register with already register user i get this:
<img width="502" alt="image" src="https://user-images.githubusercontent.com/15820737/222130169-361ccace-b243-49d0-bbc2-31b1945afe3c.png">